src/System.Globalization/tests/NumberFormatInfo/NumberFormatInfoNumberGroupSizes.cs

```diff
@@ -67,21 +67,15 @@ public void NegTest3()
 
         // TestNumberGroupSizesLocale: Verify value of property NumberGroupSizes for specific locales
         [Theory]
-        [InlineData("en-US", 3, 0)]
-        [InlineData("ur-IN", 3, 2)]
-        [ActiveIssue(3243)]
-        public void TestNumberGroupSizesLocale(string locale, int primaryGroupSize, int secondaryGroupSize)
+        [InlineData("en-US")]
+        [InlineData("ur-IN")]
+        public void TestNumberGroupSizesLocale(string locale)
```

Comment:
I'm not certain it will be possible to ensure that we can find a set of cultures that meet the following requirements:
1. They cover the different types of things being decoded (i.e. an example of each kind of calendar week rule)
2. Their culture data doesn't differ between the 5 different sets of culture data we currently support.
3. They aren't going to differ in a set of culture data we will support in the future.

Given this, I don't feel it is worth it to try to find a set of cultures that meet #1 and #2 now.  I think just finding #1 is good enough, and dealing with the differing culture data as appropriate.